### PR TITLE
Accessibility fixes for language selector, editor roles/ids

### PR DESCRIPTION
**Pull request by shakao**
fixes microsoft/pxt-microbit#3661
fixes microsoft/pxt-microbit#3652
fixes microsoft/pxt-microbit#3765
fixes microsoft/pxt-microbit#3663 
fixes microsoft/pxt-microbit#3662
fixes microsoft/pxt-microbit#3664

### CODE REVIEW

File: theme/common.less [hunk 1]
```diff
@@ -1879,6 +1879,10 @@ img.pixelart {
         justify-content: space-between;
         align-items: center;
         margin-top: 2em;
+
+        a {
```
Comment (shakao) [inline theme/common.less:1883]: fixes https://github.com/microsoft/pxt-microbit/issues/3661

File: webapp/src/gitjson.tsx [hunk 2]
```diff
@@ -755,7 +755,7 @@ class GithubComponent extends data.Component<GithubProps, GithubState> {
         const isOwner = user && user.id === githubId.owner;
         return (
             <div id="githubArea">
-                <div id="serialHeader" className="ui serialHeader">
+                <div className="ui serialHeader">
```
Comment (shakao) [inline webapp/src/gitjson.tsx:758]: fixes https://github.com/microsoft/pxt-microbit/issues/3765

File: webapp/src/lang.tsx [hunk 3]
```diff
@@ -120,7 +120,7 @@ export class LanguagePicker extends data.Component<ISettingsProps, LanguagesStat
                 closeOnEscape
             >
                 <div id="langmodal">
-                    <div id="availablelocales" className="ui cards centered" role="listbox">
+                    <div id="availablelocales" className="ui cards centered" role="list" aria-label={lf("List of available languages")}>
```
Comment (shakao) [inline webapp/src/lang.tsx:123]: fixes https://github.com/microsoft/pxt-microbit/issues/3663 (changes role to "list", children to "listitem"), fixes https://github.com/microsoft/pxt-microbit/issues/3662 (adds label)

File: webapp/src/editortoolbar.tsx [hunk 4]
```diff
@@ -321,7 +321,7 @@ export class EditorToolbar extends data.Component<ISettingsProps, {}> {
         }
         const cloudButton = <EditorToolbarButton icon={"xicon " + getCloudIcon()} className={`editortools-btn`} title={getCloudTooltip()} onButtonClick={this.cloudButtonClick} view='computer' />;
 
-        return <div id="editortools" className="ui" role="menubar" aria-label={lf("Editor toolbar")}>
+        return <div id="editortools" className="ui" role="region" aria-label={lf("Editor toolbar")}>
```
Comment (shakao) [inline webapp/src/editortoolbar.tsx:324]: fixes https://github.com/microsoft/pxt-microbit/issues/3652

File: webapp/src/toolbox.tsx [hunk 5]
```diff
@@ -825,7 +825,9 @@ export interface TreeGroupProps {
 export class TreeGroup extends data.Component<TreeGroupProps, {}> {
     renderCore() {
         const { visible } = this.props;
-        return <div role="group" style={{ backgroundPosition: '0px 0px', 'display': visible ? '' : 'none' }}>
+        if (!this.props.children) return <div />;
+
+        return <div role="tree" style={{ backgroundPosition: '0px 0px', 'display': visible ? '' : 'none' }}>
```
Comment (shakao) [inline webapp/src/toolbox.tsx:830]: fixes https://github.com/microsoft/pxt-microbit/issues/3664

**Review by jwunderl — APPROVED** [review]

**Pull request merged**